setup.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 pip install -r requirements.txt
 python3 fuel_utility.py download \
--m CGMClassic -m StorageRack -m VisitorChair \
+-m CGMClassic -m StorageRack -m VisitorChair=2 \
```

Comment:
this is required because the metadata from VisitorChair still refers to model_version==1 right?

Reply:
Updated the world to not use the problematic Visitor Chair!